client/src/App.tsx

```diff
@@ -1,31 +1,44 @@
+import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom';
+
+// Material-UI & Style
 import { MuiThemeProvider } from '@material-ui/core';
 import { theme } from './themes/theme';
-import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom';
+import './App.css';
+
+// Context
+import { AuthProvider } from './context/useAuthContext';
+import { SocketProvider } from './context/useSocketContext';
+import { SnackBarProvider } from './context/useSnackbarContext';
+
+// Components
 import Login from './pages/Login/Login';
 import Signup from './pages/SignUp/SignUp';
 import Dashboard from './pages/Dashboard/Dashboard';
-import { AuthProvider } from './context/useAuthContext';
-import { SnackBarProvider } from './context/useSnackbarContext';
+import Scheduler from './pages/Scheduler/Scheduler';
 import GoogleConnect from './components/GoogleConnect/GoogleConnect';
-import './App.css';
 
 function App(): JSX.Element {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
         <SnackBarProvider>
           <AuthProvider>
-            <Switch>
-              <Route exact path="/login" component={Login} />
-              <Route exact path="/signup" component={Signup} />
-              <Route exact path="/google" component={GoogleConnect} />
-              <Route exact path="/dashboard">
-                <Dashboard />
-              </Route>
-              <Route path="*">
-                <Redirect to="/login" />
-              </Route>
-            </Switch>
+            <SocketProvider>
```

Comment:
I don't see using Socket just yet in this project, the provider could be removed.